macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,21 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Appearance configuration keys for NSSavePanel.
+static NSString *const kPanelTitleKey = @"title";
+static NSString *const kPanelMessageKey = @"message";
+static NSString *const kOKButtonTextKey = @"prompt";
+
+// Method argument keys for NSSavePanel.
+static NSString *const kInitialDirectoryKey = @"initialDirectory";
 static NSString *const kAllowedFileTypesKey = @"allowedFileTypes";
+static NSString *const kFileNameStringValueKey = @"nameFieldStringValue";
+static NSString *const kCanCreateDirectoriesKey = @"canCreateDirectories";
```

Comment:
Do you have a specific use case for needing to configure this? It seems like we could probably just have it be on for save panels, and not add an option.

Reply:
That's a good point, I don't believe there is a particular case in which we'd want to disable this for a save panel, nor to enable it for an open panel. Done.